CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
-## 4.7.14- 2024-008-01
+## 4.7.14- 2024-08-06
```

Comment:
This should be 4.7.15 instead of 4.7.14.

Also, I guess there should be a space before the '-'. The changelog is currently not parsed correctly and thus doesn't show up on the App page in your Nextcloud instance.